extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -515,3 +529,30 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 		vscode.window.showInformationMessage(constants.MIGRATION_STARTED);
 	}
 }
+
+
+export function deepClone<T>(obj: T): T {
```

Comment:
there isn't anywhere that we can easily import this method from?

Reply:
At this point there is no such place. Should I add something in core for such methods?